include/onnxruntime/core/graph/graph.h

```diff
@@ -734,8 +734,10 @@ class Graph {
   Any graph inputs not in this list will be appended to the ordered graph input list, in the order that they were first
   used by Nodes (i.e. the order of Node creation implicitly determines the ordering).
   @remarks If the Graph was loaded from a GraphProto this has no effect.*/
-  void SetInputOrder(const std::vector<const NodeArg*> inputs) {
-    graph_input_order_ = inputs;
+  void SetInputs(const std::vector<const NodeArg*> inputs) {
```

Comment:
If the semantics of this method is being changed, it would help to update the preceding documentation as well. E.g., the "any graph inputs not in this list …" part. Is that being changed?

Reply:
yes, I'll update the comments definitely. sending this out for folks, specifically @liqunfu to see whether there're any concerns from users. 

I'll also make these two APIs public.